.travis.yml

```diff
@@ -109,12 +109,28 @@ jobs:
           branch: master
           repo: denoland/deno
         skip-cleanup: true
+    
+    - name: "cargo release linux arm64"
+      os: linux
+      dist: xenial
+      after_success:
+        - gzip -c $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release/deno > target/release/deno_${TRAVIS_OS_NAME}_arm64.gz
+      script:
+      - rustup target add aarch64-unknown-linux-gnu
+      - sudo apt update
+      - sudo apt -yq --no-install-suggests --no-install-recommends install g++-5-aarch64-linux-gnu gcc-5-aarch64-linux-gnu g++-5-multilib libc6-arm64-cross
+      - build/linux/sysroot_scripts/install-sysroot.py --arch=arm64
+      - export DENO_BUILD_ARGS="target_cpu=\"arm64\" v8_target_cpu=\"arm64\""
+      - export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - export CC_aarch64_unknown_linux_gnu="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - ./tools/setup.py
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --locked
```

Comment:
Please add a very simple test after the build:
```
- $CARGO_TARGET_DIR/aarch64-unknown-linux-gnu/release/deno tests/002_hello.ts
```

Reply:
Working on a good way to virtualize tests with qemu right now.

Reply:
ah right. sorry - I forgot that it needs to be emulated.